src/core/a-entity.js

```diff
@@ -301,14 +302,52 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  getMixedInComponents: {
+    value: function () {
+      var mixinEls = this.mixinEls;
+      var i;
+      var components = [];
+      for (i = 0; i < mixinEls.length; ++i) {
+        var keys = Object.keys(mixinEls[i].attrs);
```

Comment:
Don't need the intermediate var. Object.keys.forEach is a pretty common pattern.